web/src/lib/components/asset-viewer/asset-viewer.svelte

```diff
@@ -756,9 +756,9 @@
   {#if isShowAlbumPicker}
     <AlbumSelectionModal
       shared={addToSharedAlbum}
-      on:newAlbum={({ detail }) => handleAddToNewAlbum(detail)}
-      on:album={({ detail }) => handleAddToAlbum(detail)}
-      on:close={() => (isShowAlbumPicker = false)}
+      onNewAlbum={handleAddToNewAlbum}
```

Comment:
You handlers have a return signature of void, but the actual handler if often async. If svelte-check does catch that, it could be a problem.

Ideally, you would need to make the handler like this, and then you'd be forced to await it.
```ts
export let onNewAlbum: (search: string) => Promise<unknown>;
```

Reply:
Yeah, I'm trying to figure out why `svelte-check`/`eslint` aren't catching it. That's why I sent the eslint PR (https://github.com/immich-app/immich/pull/7382), which I expected would catch this. Unfortunately, it's still not. Perhaps it's an issue with `eslint-plugin-svelte` since it's in the Svelte markup portion of the template. I'm leaving this broken and in draft mode until I track it down as I'm more concerned with having an automated check to avoid the error than fixing this particular instance of it. Next step on that journey is here: https://github.com/immich-app/immich/pull/7471

Reply:
I filed an issue in the `eslint-plugin-svelte` repo: https://github.com/sveltejs/eslint-plugin-svelte/issues/691

I'm going to close this PR until that's implemented as it's too hard to keep everything straight without the linter's help